pkgs/development/interpreters/php/default.nix

```diff
@@ -113,8 +114,9 @@ let
       configureFlags = [
         "--with-config-file-scan-dir=/etc/php.d"
       ]
-      ++ optional (versionOlder version "7.3") "--with-pcre-regex=${pcre.dev} PCRE_LIBDIR=${pcre}"
-      ++ optional (versionAtLeast version "7.3") "--with-pcre-regex=${pcre2.dev} PCRE_LIBDIR=${pcre2}"
+      ++ optional ((versionOlder version "7.3") && (!versionAtLeast version "7.4")) "--with-pcre-regex=${pcre.dev} PCRE_LIBDIR=${pcre}"
```

Comment:
`v < 7.3 && !(v ≥ 7.4) = v < 7.3 && v < 7.4 = v < 7.3`

```suggestion
      ++ optional (versionOlder version "7.3") "--with-pcre-regex=${pcre.dev} PCRE_LIBDIR=${pcre}"
```

Reply:
In version 7.3 uses the parameter `"--with-pcre-regex"`, and in the version 7.4 `"--with-external-pcre"`

Reply:
Any reason why not the bundled one?

Reply:
See https://fedoraproject.org/wiki/Bundled_Libraries

Reply:
Well, the versions covered by `< 7.3` are a subset of those covered by `!(v ≥ 7.4)`, so the second conjunct is not really needed.

Reply:
fixed, thanks!